src/gen/js/service.js.template

```diff
@@ -258,7 +258,10 @@ function formatServiceError(response, data, options) {
 }
 
 function processError(req, error) {
-	return Promise.resolve({ res: { raw: {}, data: error, error: true } });
+  const { processError } = options;;
```

Comment:
Can you also use tabs instead of spaces as indent. We use that to parse and swap out indentation type as per configuration.